x-pack/plugins/monitoring/public/components/apm/instance/status.js

```diff
@@ -22,35 +23,58 @@ export function Status({ stats }) {
 
   const metrics = [
     {
-      label: 'Name',
+      label: intl.formatMessage({
+        id: 'xpack.monitoring.apm.instance.status.nameLabel',
+        defaultMessage: 'Name',
+      }),
       value: name,
       dataTestSubj: 'name'
     },
     {
-      label: 'Output',
+      label: intl.formatMessage({
+        id: 'xpack.monitoring.apm.instance.status.outputLabel',
+        defaultMessage: 'Output',
+      }),
       value: output,
       dataTestSubj: 'output'
     },
     {
-      label: 'Version',
+      label: intl.formatMessage({
+        id: 'xpack.monitoring.apm.instance.status.versionLabel',
+        defaultMessage: 'Version',
+      }),
       value: version,
       dataTestSubj: 'version'
     },
     {
-      label: 'Uptime',
+      label: intl.formatMessage({
+        id: 'xpack.monitoring.apm.instance.status.uptimeLabel',
+        defaultMessage: 'Uptime',
+      }),
       value: formatMetric(uptime, 'time_since'),
       dataTestSubj: 'uptime'
     },
     {
-      label: 'Last Event',
+      label: intl.formatMessage({
+        id: 'xpack.monitoring.apm.instance.status.lastEventLabel',
+        defaultMessage: 'Last Event',
+      }),
       value: formatTimestampToDuration(+moment(timeOfLastEvent), 'since') + ' ago',
```

Comment:
This should be translated as well. The similar case is here https://github.com/elastic/kibana/pull/24736/files#diff-5624fed0f1f9a5d7aad781e974e5420cR78 .